HHS/Head-Start-TTADP
# [TTAHUB-1151] add training vs technical assistance filter

## Description of change
Add filter for TTA type to the landing page and the regional dashboard
Note that this filter's query working 100% depends on the database cleanup being worked on in [this ticket](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1275).

The jira for the queue also necessitated a small change to the AR form language, which necessitated a small change to the e2e tests.

I took advantage of our typescript support and added the first new typescript file into our codebase - I ran into a few issues with the linter which I detailed a little more within the comments on this PR. You'll have to update your locally dependencies to see the benefit.

I also spent a bunch of time trying to figure out issues with the e2e tests (the issue that Jon eventually fixed) but this led me to do some cleanup on our tests, consolidating some of the files that contained one test into other suites. I think this is a little cleaner and might help fight the sprawl.

## How to test
Add a filter for "is" & "is not" each of the types. Spot check this info in the database. Confirm everything functions as desired.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1151


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
